test/testrunner.py

```diff
@@ -424,18 +424,19 @@ def main():
         metrics_utils.send_test_duration_metrics(start_time)
 
     elif specific_test_type == "sagemaker-local":
-        if "neuron" in dlc_images:
-            LOGGER.info(f"Skipping sagemaker tests because Neuron is not yet supported on SM. Images: {dlc_images}")
+        sm_local_to_skip = {
+            "habana": "Skipping SM tests because SM does not yet support Habana",
+            "neuron": "Skipping - there are no local mode tests for Neuron",
+            "huggingface-tensorflow-training": "Skipping - there are no local mode tests for HF TF training"
+        }
+
+        for skip_condition, reason in sm_local_to_skip.items():
+            LOGGER.info(f"{reason}. Images: {dlc_images}")
```

Comment:
There needs to be an if condition here - if skip_condition in dlc_images - right now this is skipping everything